client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -88,12 +88,12 @@ class VerificationCodeForm extends Component {
 		} = this.props;
 		const isError = !! twoFactorAuthRequestError;
 
-		let helpText = translate( 'Type in the code generated by your Authenticator mobile application.' );
+		let helpText = translate( 'Enter the code generated by your Authenticator mobile application.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Please enter the verification code generated by your Authenticator mobile application.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).